osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/domain/DbEncryptionKeyReference.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.secretmanagement.application.domain;
+
+import java.util.Date;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.opensmartgridplatform.shared.security.EncryptionProviderType;
+
+@Entity
+@Table(name = "encryption_key_reference")
+@Getter
+@Setter
+public class DbEncryptionKeyReference {
```

Comment:
why is this class prepended with Db (I understand why, but it is not standard to the rest of the app).

Consider changing to EncryptionKeyReference, then you also do not need the name attribute of @Table